client/src/Components/Layouts/Customers/index.jsx

```diff
@@ -31,20 +32,37 @@ export default class Customers extends Component {
         visible: false,
         loading: true,
         customersPage: {
-            deleteVisibility: false,
-            id: ''
+            delete: {
+                deleteVisibility: false,
+                information: [],
+                id: ''
+            },
+            edit: {
+                editVisibility: false,
+                id: '',
+                information: [],
+            }
+
         },
+        error:''
     }
     componentDidMount() {
-        fetch('api/v1/customers').then(res => res.json())
+        fetch('/api/v1/customers').then(res => {
+            console.log(res)
+            if(res.status===200){
+           return res.json()}else{
+               console.log(res)
+               this.setState({error:res})
```

Comment:
@Jamalat-shamallakh I think you might want to throw an error here instead (`throw new Error(res)`) so that the error is handled on line 65.  This looks good but I think if it doesn't actually `throw`, the promise will keep going into the next then() block which might be bad.